astroquery/ipac/nexsci/nasa_exoplanet_archive/core.py

```diff
@@ -387,18 +391,36 @@ def query_aliases(self, object_name, *, cache=None):
         response : list
             A list of aliases found for the object name. The default name will be listed first.
         """
-        return list(
-            self.query_criteria(
-                "aliastable", objname=object_name.strip(), cache=cache, format="csv"
-            )["aliasdis"]
-        )
+        url = requests.get("https://exoplanetarchive.ipac.caltech.edu/cgi-bin/Lookup/nph-aliaslookup.py?objname="+object_name)
```

Comment:
As you added the base URL for the lookup to the config, please us it from there.